oidc-ui/.env.production

```
@@ -0,0 +1 @@
+VITE_API_URL=https://localhost:8090
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)